Vue InstantSearch/media/src/App.vue

```diff
@@ -43,7 +43,7 @@
             <template v-slot="{ items }">
               <div class="movies">
                 <ais-state-results>
-                  <template v-slot="{ query, hits }">
+                  <template v-slot="{ results: { query , hits } }">
                     <p
                       class="movies-no-results"
                       v-if="hits.length === 0"
```

Comment:
because of this v-if without else, if there *are* hits, it shows the fallback information. Maybe add an empty div in `v-else`?

Reply:
Done (I think) ✅